fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -524,6 +526,17 @@ identifierList
     : '(' identifier (',' identifier)* ')'
     ;
 
+identifierOrString
+    : identifier        # identifierOfIdentifierOrString
+    | string            # stringOfIdentifierOrString
+    ;
+
+userIdentifier
```

Comment:
user is not only a identifier, I think just calling “user” is a better name

Reply:
OK